### DISCUSSION TITLE
Re-ignore errors automatically, possible?

### DISCUSSION THREAD

**Discussion by davidmasp** [nextflow-io/nextflow#2270] [Q&A]
Let's say I have a process which "fails" 50% of the time and I ignore them. When I want to resume the pipeline, those ignored jobs re-run because files are not there. Is there a way I am missing to manage this situation and "skip" the execution of the ignored processes?

**Comment by pditommaso** [comment]
Basically, you want to ignore them also on resume. Not there's no way to do that.
